hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/impl/BackupManager.java

```diff
@@ -295,6 +295,11 @@ public void setBackupInfo(BackupInfo backupInfo) {
           .withRootDir(backup.getBackupRootDir()).withTableList(backup.getTableNames())
           .withStartTime(backup.getStartTs()).withCompleteTime(backup.getCompleteTs()).build();
 
+      // Not required if root dir is not same.
+      if (!image.getRootDir().equals(backupInfo.getBackupRootDir())) {
```

Comment:
Maybe add a because here? (I say this because this change looks fine but looking at this and looking at the surrounding code, I'm not clear what is going on -- 'image', 'ancestor' -- and I'm not that inclined to spend much time doing research because as you know, I'm not that mad about the backup feature!). Thanks.